alibi_detect/od/backends/keops/knn.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import torch
+from pykeops.torch import LazyTensor
+
+
+def cdist(X, Y):
```

Comment:
Similar remarks about lowercasing throughout the script as in the `torch` version.